pages/join.js

```diff
@@ -9,44 +9,41 @@ import RegistrationForm from 'components/RegistrationForm/RegistrationForm';
 
 const profileUpdateURL = '/profile/update';
 
-class Join extends React.Component {
-  static propTypes = {
-    router: object.isRequired,
-  };
-
-  componentDidMount() {
-    const { router } = this.props;
-
+const Join = props => {
```

Comment:
Can you destructure props here?

Reply:
done

Reply:
I meant: `function Join({ router }) {` then you can delete line 14